calculation/value.go

```diff
@@ -16,18 +18,40 @@ const (
 	StringType
 )
 
+var (
+	nullBytes = []byte("null")
```

Comment:
Do we want to define this somewhere in a shared location?

Reply:
Hm a bit overkill given this is only used in two places in two separate packages.